server.js

```diff
@@ -6,12 +8,11 @@
 
 const { createServer } = require('http')
 const { parse } = require('url')
-const _ = require('force-ssl-heroku')
 const next = require('next')
 const querystring = require('querystring')
 
 const dev = process.env.NODE_ENV !== 'production'
-const app = next({ dev })
+const app = next({ dev, crossOrigin: 'anonymous' })
```

Comment:
Is this the same as setting a config option for that?

```js
// next.config.js
module.exports = {
  crossOrigin: 'anonymous',
}
```

which seems to be the official rec, per https://github.com/zeit/next.js#cdn-support-with-asset-prefix

Reply:
Ooops, my bad. Forgot to remove it from `server.js`, it is not needed here anymore.

Reply:
OK 🙂